services/src/main/java/org/keycloak/authentication/requiredactions/VerifyUserProfile.java

```diff
@@ -46,11 +41,15 @@
 /**
  * @author <a href="mailto:psilva@redhat.com">Pedro Igor</a>
  */
-public class VerifyUserProfile implements RequiredActionProvider, RequiredActionFactory, DisplayTypeRequiredActionFactory {
+public class VerifyUserProfile extends UpdateProfile {
 
     @Override
     public InitiatedActionSupport initiatedActionSupport() {
-        return InitiatedActionSupport.SUPPORTED;
+        return InitiatedActionSupport.NOT_SUPPORTED;
+    }
+    
+    protected UserModel.RequiredAction getResponseAction(){
```

Comment:
Nitpick. Could you add the `@Override` annotation?

Reply:
sure

Reply:
updated